backend/infrahub/events/schema_action.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any
+
+from infrahub.message_bus import InfrahubMessage
+from infrahub.message_bus.messages.refresh_registry_branches import RefreshRegistryBranches
+
+from .models import InfrahubBranchEvent
+
+
+class SchemaUpdatedEvent(InfrahubBranchEvent):
+    """Event generated when a node has been mutated"""
+
+    def get_name(self) -> str:
+        return f"{self.get_event_namespace()}.schema.update"
+
+    def get_resource(self) -> dict[str, str]:
+        return {
+            "prefect.resource.id": f"infrahub.schema_branch.{self.branch}",
+            "infrahub.branch.name": self.branch,
+        }
+
+    def get_payload(self) -> dict[str, Any]:
+        return {"branch": self.branch}
+
+    def get_messages(self) -> list[InfrahubMessage]:
+        return [
```

Comment:
to be consistent with the other events, I think we should keep sending the original message for now
I understand it's not 100% working yet and we need to do some change

Reply:
> or if we should have an initial check that waits for the hashes to be updated on all workers.

Yes i think that would be good to have a task for that, we could easily reuse that in multiple places

Reply:
Ok, I'll look at that in the next PR as it's a bit more generic and not strictly related to this one.